docs/guide.pug

```diff
@@ -201,13 +201,11 @@ block content
 
     ```javascript
       // Assign a function to the "statics" object of our animalSchema
-      animalSchema.statics.findByName = function(name) {
-        return this.find({ name: new RegExp(name, 'i') });
-      };
+      animalSchema.statics.findByName = name => (
+        this.find({ name: new RegExp(name, 'i') });
+       )
       // Or, equivalently, you can call `animalSchema.static()`.
-      animalSchema.static('findByBreed', function(breed) {
-        return this.find({ breed });
-      });
+      animalSchema.static('findByBreed', breed => this.find({ breed }) )
```

Comment:
This change doesn't work because the static function uses `this`, which [arrow functions](https://masteringjs.io/tutorials/fundamentals/arrow) mess up.